apps/sms/js/thread_list_ui.js

```diff
@@ -402,10 +403,18 @@ var ThreadListUI = {
         thread.unreadCount = isRead ? 0 : 1;
         this.mark(thread.id, isRead ? 'read' : 'unread');
```

Comment:
because this is "thread.unreadCount = ..;" and "this.mark(..." are both part of the "markReadUnread" processus, right ?